operations/companies.md

```diff
@@ -204,36 +286,36 @@ Adds a new company. Note this operation supports [Portfolio Access Tokens](../gu
 
 ```javascript
 {
-    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-    "Client": "Sample Client 1.0.0",
-    "ChainId": "1df21f06-0cfc-4960-9c58-a3bf1261663e",
-    "Name": "Mews",
-    "Options": {
-        "Invoiceable": true,
-        "AddFeesToInvoices": true
-    },
-    "MotherCompanyId": null,
-    "Identifier": null,
-    "TaxIdentifier": null,
-    "AdditionalTaxIdentifier": null,
-    "BillingCode": null,
-    "AccountingCode": null,
-    "Address": null,
-    "InvoiceDueInterval": "P2DT23H",
-    "Telephone": "111-222-333",
-    "ContacPerson": "SamplePerson",
-    "Contact": "ContactInfo",
-    "Notes": "Note1",
-    "Iata": "PAO",
-    "Department": "Sales",
-    "DunsNumber": "987654321",
-    "CreditRating": {
-        "Basic": "CreditOk"
-    },
-    "ExternalIdentifier": "1234",
-    "ReferenceIdentifier": "a58ff7cb-77e3-495a-bd61-aecf00a3f19d",
-    "WebsiteUrl": "https://www.mews.com"
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
```

Comment:
ToDo: figure out how the method description is generated `Adds a new company. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).`